FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Common/Metrics/ErrorType.cs

```diff
@@ -7,6 +7,11 @@ namespace Microsoft.Health.Fhir.Synapse.Common.Metrics
 {
     public class ErrorType
     {
+        /// <summary>
+        /// A metric type for errors that occur when creating job.
+        /// </summary>
+        public static string CreateJobError => nameof(JobExecutionError);
```

Comment:
The `CreateJobError` corresponds to `JobExecutionError`, is this by design or a typo?

Reply:
Thanks, fixed it, it's a typo.